packages/myst-to-jats/src/index.ts

```diff
@@ -27,9 +27,14 @@ import type { SupplementaryMaterial } from './transforms/containers.js';
 import type { Section } from './transforms/sections.js';
 import { sectionAttrsFromBlock } from './transforms/sections.js';
 import { inlineExpression } from './inlineExpression.js';
+import { notebookArticleSuffix, slugSuffix } from './utils.js';
 
 type TableCell = SpecTableCell & { colspan?: number; rowspan?: number; width?: number };
 
+function escapeForXML(text: string) {
```

Comment:
Pretty simple function, but & and < are the only two characters that must be escaped in text.

Reply:
Nice -- that is awesome.